Assets/MixedRealityToolkit.Tools/ExtensionServiceCreator/ExtensionServiceCreator.cs

```diff
@@ -273,7 +275,7 @@ public string SampleCode
 
         #region paths
 
-        private string ExtensionsFolder => MixedRealityToolkitFiles.MapModulePath(MixedRealityToolkitModuleType.Extensions);
+        private string ExtensionsFolder => Path.Combine("Assets", DefaultGeneratedFolderName, DefaultExtensionsFolderName);
```

Comment:
Is this the default, or the only option? I don't remember if this is configurable or not

Reply:
This is the default! It doesn't have the word default in it, but if there's some persisted session key (i.e. the place where I added that weird false &&) it uses what the consumer chose/overwrote.